LICENSE.txt

```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    {{project}}  Copyright (C) {{year}}  {{fullname}}
+    Copyright (C) <year> <name of author>
```

Comment:
Cool. Then we need to change https://github.com/Islandora/islandora_bagit/pull/43/files#diff-61e0bdf7e1b43c5c93d9488b22e04170R667 and other {} here. Thanks!